config/environments/production.rb

```diff
@@ -2,6 +2,10 @@
 uncomment_lines "config/environments/production.rb", /config\.active_job/
 uncomment_lines "config/environments/production.rb", /raise_delivery_errors =/
 gsub_file "config/environments/production.rb", ":resque", ":sidekiq"
+gsub_file "config/environments/production.rb", " (and separate queues per environment)", ""
+gsub_file "config/environments/production.rb",
+          /queue_name_prefix = .*$/,
+          "queue_name_prefix = nil # Not supported by sidekiq"
```

Comment:
Queue names prefixes do not not fully isolate one Sidekiq instance from others sharing the same Redis, because Sidekiq has special keys it uses for keeping track of other metadata. These keys aren't "queues", so they don't get prefixed, and so multiple apps would clobber each other.